src/VisualStudio/TestUtilities/Interop/Kernel32.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace Roslyn.VisualStudio.Test.Utilities.Interop
+{
+    internal static class Kernel32
+    {
+        [DllImport("Kernel32.dll", CallingConvention = CallingConvention.Winapi, EntryPoint = "GetCurrentThreadId", PreserveSig = true, SetLastError = false)]
+        public static extern uint GetCurrentThreadId(
+        );
```

Comment:
Hmm, I normally only do that when there are parameters that need explicit `In`/`Out`/`MarshalAs` attributes.

Fixing.